admin/InitDb.pl

```diff
@@ -277,6 +278,7 @@ sub CreateRelations
 
     RunSQLScript($DB, "CreateTables.sql", "Creating tables ...");
     RunSQLScript($DB, "caa/CreateTables.sql", "Creating tables ...");
+    RunSQLScript($DB, "eaa/CreateTables.sql", "Creating tables ...");
```

Comment:
Wouldn't it make sense for this (and caa and report) to specify which tables, since the others do?

Reply:
Added in ac64e7bdf65c6de41bf24d36daae8a27570dd75f